web/pdf_viewer.js

```diff
@@ -1080,10 +1091,18 @@ class PDFViewer {
     );
   }
 
-  _setScaleUpdatePages(newScale, newValue, noScroll = false, preset = false) {
+  _setScaleUpdatePages(
+    newScale,
+    newValue,
+    { noScroll = false, preset = false, delay = -1, update = false }
+  ) {
     this._currentScaleValue = newValue.toString();
 
     if (this.#isSameScale(newScale)) {
+      if (update) {
+        this.refresh({ scale: newScale });
+        return;
+      }
```

Comment:
Based on a quick whole-words search I'm not finding anything that ever sets the `update`-option!?